pkg/tool/helmclient/helmclient.go

```diff
@@ -31,21 +31,59 @@ import (
 	"helm.sh/helm/v3/pkg/downloader"
 	"helm.sh/helm/v3/pkg/release"
 	"helm.sh/helm/v3/pkg/releaseutil"
+	"helm.sh/helm/v3/pkg/storage"
 	"helm.sh/helm/v3/pkg/storage/driver"
 	"helm.sh/helm/v3/pkg/strvals"
 	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/yaml"
 
+	"github.com/koderover/zadig/pkg/microservice/aslan/config"
+	"github.com/koderover/zadig/pkg/microservice/aslan/core/common/service/kube"
+	kubeclient "github.com/koderover/zadig/pkg/shared/kube/client"
+	"github.com/koderover/zadig/pkg/tool/kube/updater"
 	"github.com/koderover/zadig/pkg/tool/log"
 	yamlutil "github.com/koderover/zadig/pkg/util/yaml"
 )
 
 type HelmClient struct {
 	*hc.HelmClient
+	kubeClient client.Client
+	Namespace  string
+}
+
+func NewClientFromNamespace(clusterID, namespace string) (hc.Client, error) {
+	restConfig, err := kube.GetRESTConfig(clusterID)
```

Comment:
It may be better to use `"github.com/koderover/zadig/pkg/shared/kube/client`" directly.
The encapsulation of the common library in `aslan` has little effect. It is suggested to reduce the dependence on such encapsulation gradually.